easybuild/easyconfigs/n/ncregrid/ncregrid-1.4b-goolf-1.4.10.eb

```diff
@@ -0,0 +1,37 @@
+##This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+#
+##name = 'ncregrid'
+
+version = '1.4b'
+
+homepage = 'http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/'
+description = """ncregrid is a tool (written in FORTRAN-90) for data transfer of gridded 2- and 3-dimensional (spatial) geophysical/geochemical scalar fields between grids of different resolutions"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = ['http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/']
+sources = [('%(namelower)s.tar.Z', 'tar -xzf %s')]
+
+configopts = "F90=gfortran F90FLAGS='-cpp' NC_INC=$EBROOTNETCDFMINFORTRAN/include NC_LIB=$EBROOTNETCDFMINFORTRAN/lib"
```

Comment:
@wpoely86: F90FLAGS is a mandatory option for configuring ncregrid...I will remove only F90 which is optional